commands/azureCommands/acr-build.ts

```diff
@@ -45,11 +37,11 @@ export async function queueBuild(dockerFileUri?: vscode.Uri): Promise<void> {
     } else {
         folder = await (<any>vscode).window.showWorkspaceFolderPick();
     }
-    const dockerItem = await resolveDockerFileItem(folder, dockerFileUri);
+    const dockerItem = await resolveFileItem(folder, dockerFileUri, "DockerFile");
     const sourceLocation: string = folder.uri.path;
-    const tarFilePath = getTempSourceArchivePath();
+    const tarFilePath = getTempSourceArchivePath(status);
 
-    const uploadedSourceLocation = await uploadSourceCode(client, registry.name, resourceGroupName, sourceLocation, tarFilePath, folder);
+    const uploadedSourceLocation = await uploadSourceCode(status, client, registry.name, resourceGroupName, sourceLocation, tarFilePath, folder);
     status.appendLine("Uploaded Source Code to " + tarFilePath);
```

Comment:
Can `status.appendLine` be moved into `uploadSourceCode`?

Reply:
done